src/com/google/javascript/jscomp/CommandLineRunner.java

```diff
@@ -1191,15 +1193,16 @@ protected Compiler createCompiler() {
   }
 
   @Override
-  protected List<SourceFile> createExterns() throws FlagUsageException,
-      IOException {
-    List<SourceFile> externs = super.createExterns();
-    if (flags.useOnlyCustomExterns || isInTestMode()) {
+  protected List<SourceFile> createExterns(CompilerOptions options)
+      throws FlagUsageException, IOException {
+    List<SourceFile> externs = super.createExterns(options);
+    if (isInTestMode()) {
       return externs;
     } else {
-      List<SourceFile> defaultExterns = getDefaultExterns();
-      defaultExterns.addAll(externs);
-      return defaultExterns;
+      List<SourceFile> builtinExterns =
+          getBuiltinExterns(options);
```

Comment:
Oh just to do

    List<SourceFile> builtinExterns = getBuiltinExterns(options);